### PR TITLE
LW-612 iOS: Stash w/o nominator/validator

### PR DESCRIPTION
**Pull request by ERussel**
- make sure that controller exists locally for settings validators
- add implementation for change validators flow

### CODE REVIEW

File: fearless/Modules/Staking/StakingConfirm/StakingConfirmViewFactory.swift [hunk 1]
```diff
@@ -102,6 +132,49 @@ final class StakingConfirmViewFactory: StakingConfirmViewFactoryProtocol {
...
-                                                 nomination: nomation)
+                                                 nomination: nomination)
+    }
+
+    static private func createChangeTargetsInteractor(_ nomination: PreparedNomination<ExistingBonding>,
+                                                      connection: JSONRPCEngine,
+                                                      keystore: KeystoreProtocol,
+                                                      assetId: WalletAssetId,
+                                                      networkSettings: ConnectionItem)
+    -> ChangeTargetsConfirmInteractor? {
```
Comment (ef1rspb) [inline fearless/Modules/Staking/StakingConfirm/StakingConfirmViewFactory.swift:143]: or StakingBaseConfirmInteractor?

File: fearless/Modules/Staking/StakingMain/StakingMainInteractor+InputProtocol.swift [hunk 2]
```diff
@@ -21,6 +22,23 @@ extension StakingMainInteractor: StakingMainInteractorInputProtocol {
 
         applicationHandler.delegate = self
     }
+
+    func fetchController(for address: String) {
```
Comment (ef1rspb) [inline fearless/Modules/Staking/StakingMain/StakingMainInteractor+InputProtocol.swift:26]: AccountAddress?

Comment (ERussel) [reply fearless/Modules/Staking/StakingMain/StakingMainInteractor+InputProtocol.swift]: fixed

File: fearless/Modules/Staking/StakingConfirm/StakingConfirmViewFactory.swift (hunk 1 above)
Comment (ERussel) [reply fearless/Modules/Staking/StakingConfirm/StakingConfirmViewFactory.swift]: fixed

**Review by ef1rspb — APPROVED** [review]

**Pull request merged**